doc/manual/citing_aspect.bib

```diff
@@ -1665,3 +1665,48 @@ @Article{se-14-389-2023
 URL = {https://se.copernicus.org/articles/14/389/2023/},
 DOI = {10.5194/se-14-389-2023}
 }
+
+@mastersthesis { 2023limitsofte,
+	title = {Limits of tectonic reactivation on Mars using Earth analogue analysis and numerical modeling},
+	institution = {Mississippi State University},
+	month = {May},
```

Comment:
The CI errors out:
```
Biber warning: [1497] Utils.pm:293> WARN - month field 'May' in entry '2023limitsofte' is not an integer - this will probably not sort properly.
```
I suspect that you need to write this as follows:
```suggestion
	month = may,
```
Don't ask me why.